R/pool.R

```diff
@@ -267,36 +338,50 @@ pool_bootstrap_percentile <- function(est, conf.level, alternative) {
 }
 
 
-#' Title
+#' Bootstrap Pooling via normal approximation
 #' @description
-#' TODO
-#' @param est TODO
-#' @param conf.level TODO
-#' @param alternative TODO
+#' Creates a point estimate, confidence interval and p-values using
```

Comment:
"Creates a point estimate, confidence interval and p-values" -> "Get point estimate, confidence interval and p-value"

Reply:
done